src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.controller;
+
+import javax.validation.Valid;
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final UserResponseMapper userResponseMapper;
```

Comment:
<img width="904" alt="image" src="https://user-images.githubusercontent.com/81781479/204826095-d06e2699-971c-4553-bb84-bc1f5786801b.png">

Reply:
Occasionally left it unattended. Of course.